src/brasil/gov/portal/browser/album/albuns.py

```diff
@@ -283,8 +283,7 @@ def _get_brains(self, data_type=None):
     def _get_items(self):
```

Comment:
IMO, this can be further simplified just accessing the list of objects on the container using something similar to what we do in collective.nitf: https://github.com/collective/collective.nitf/blob/2.1b4/src/collective/nitf/content.py#L27-L30